File: Data Structures & Algorithms/duplicate-integer/submission-4.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        if len(nums) == 0:
            return False
        for i in nums:
         count = 0
         for j in nums:
              if i == j:
                count+=1
                if count > 1:
                    return True
        return False

File: Data Structures & Algorithms/top-k-elements-in-list/submission-4.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int, int> frequency;
        vector<int> solution;
        for (int num : nums)
            frequency[num]++;
        vector<int> frequencyArray;
        for(auto element : frequency)
            frequencyArray.push_back(element.second);
        sort(frequencyArray.begin(), frequencyArray.end(), greater<int>());
        for (int i = 0; i < k; i++) {
            for(auto &element : frequency){
                if(element.second == frequencyArray[i]) {
                    solution.push_back(element.first);
                    element.second = -1;
                    break;
                }
            }
        }
        return solution;
    }
};
